projects/clinician-insights/daily-feedback.py

```diff
@@ -0,0 +1,413 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+description: provide daily feedback to clinicians
+version: 0.0.1
+created: 2018-08-01
+author: Ed Nykaza
+dependencies:
+    * requires tidepool-analytics-env (see readme for instructions)
+    * requires a clinician or study username (email) and password
+    * requires tidals (tidepool data analytics tools)
+license: BSD-2-Clause
+"""
+
+
+# %% REQUIRED LIBRARIES
+import pandas as pd
+import numpy as np
+import os
+import sys
+import requests
+import json
+import argparse
+from pytz import timezone
+from datetime import timedelta
+import datetime as dt
+import subprocess as sub
+sys.path.insert(0, os.path.abspath(os.path.join(__file__, "..", "..", "..")))
+import tidals as td
+
+
+# %% USER INPUTS
+codeDescription = "Provide daily feedback to clinicians"
+
+parser = argparse.ArgumentParser(description=codeDescription)
+
+parser.add_argument("-d",
+                    "--date-stamp",
+                    dest="dateStamp",
+                    default=dt.datetime.now().strftime("%Y-%m-%d"),
+                    help="date of the daily report, defaults to current date")
+
+parser.add_argument("-e",
+                    "--email",
+                    dest="email",
+                    default=np.nan,
+                    help="the email address of the master clinicain or study account")
+
+parser.add_argument("-p",
```

Comment:
In general, it's better practice to get a password using `getpass`, or an environment variable.
You could even do both, where if the environment variable is set, that gets used, otherwise use `getpass`.
When using an environment variable, you can still launch the process in one line using something like this:
`PASSWORD=somepassword python3 /path/to/script`.
Passing in a password as a command line arg makes it visible in the process arguments when running `ps`, meaning that any non-privileged user that gets access to the machine could find out the password.

Reply:
great feedback...thank you...I will make this change